src/plat/maaxboard/overlay-maaxboard.dts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ * Copyright 2022, Capgemini Engineering
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ */
+
+/ {
+	chosen {
+		seL4,elfloader-devices =
+			"serial0",
+			&{/psci};
+
+		seL4,kernel-devices =
+			"serial0",
+			&{/soc@0/interrupt-controller@38800000},
+			&{/timer};
+	};
+
+	/* Redefine USB to use dwc3 driver; modify usb@38x00000 entries from maaxboard.dts.
```

Comment:
Can you make this comment a bit more verbose, why the "dwc3" driver is preferred here?

Reply:
Added a further comment about dwc3.